.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: trusty
 language: rust
 rust:
-  - 1.17.0
+  - stable
```

Comment:
I think we should be tracking the version Gecko uses here, which is not always `stable`

Reply:
@kvark see https://groups.google.com/d/msg/mozilla.dev.platform/IL6OrN_5ifg/TaUcsWvYBAAJ - Gecko is tracking stable rust, but with a two week delay. Unless we want to keep bumping this every 6 weeks I think it's better to just leave it at `stable`. Considering we're now at 1.19 clearly nobody remembered to update this for the last two rust version bumps, I don't think it's reasonable to expect people to remember to update this for each new rust release.

Reply:
So you are willing to allow every 2/6 weeks (so, 33% of time) for WR to base on the Rust version that has more features than Gecko sees? I'd rather keep the current version fixed until someone  is bored or want to use specific feature that they can confirm Gecko's Rust supports.

Reply:
Ok, fair enough. I've updated the patch to just bump it from 1.17 to 1.19.